Sources/MapboxCoreNavigation/NavigationSettings.swift

```diff
@@ -73,17 +115,6 @@ public class NavigationSettings {
      */
     public static let shared = NavigationSettings()
     
-    /// Returns a reflection of this class excluding the `properties` variable.
-    lazy var properties: [Mirror.Child] = {
```

Comment:
Why has this been removed?

Reply:
It is not used anywhere. Do I miss something?